src/Tile/ProductTile.js

```diff
@@ -1,37 +1,14 @@
-import classnames from 'classnames';
 import ProductTileContent from './_ProductTileContent';
 import ProductTileMedia from './_ProductTileMedia';
-import PropTypes from 'prop-types';
 import React from 'react';
+import Tile from './Tile';
 
 const ProductTile = props => {
-    const { disabled, children, className, ...rest } = props;
-
-    const tileProductClasses = classnames(
-        'fd-product-tile',
-        {
-            'is-disabled': disabled
-        },
-        className
-    );
-
-    return (
-        <div
-            {...rest}
-            className={tileProductClasses}>
-            {children}
-        </div>
-    );
+    return (<Tile productTile {...props} />);
 };
 
 ProductTile.displayName = 'ProductTile';
 
-ProductTile.propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-    disabled: PropTypes.bool
-};
-
 ProductTile.Content = ProductTileContent;
```

Comment:
This can be changed to `ProductTile.Content = TileContent;` (see other related comments)